example/src/main/java/org/eclipse/hono/vertx/example/base/HonoConsumerBase.java

```diff
@@ -144,14 +147,39 @@ private void printMessage(final String tenantId, final Message msg, final String
     /**
      * Handler method for a <em>device ready for command</em> notification (by an explicit event or contained implicit in
      * another message).
+     * <p>
+     * The code creates a simple command in JSON
      *
-     * @param timeUntilDisconnectNotification The notification containing the tenantId, deviceId and the Instant (that
-     *                                         defines until when this notification is valid). See {@link TimeUntilDisconnectNotification}.
+     * @param notification The notification containing the tenantId, deviceId and the Instant (that
+     *                     defines until when this notification is valid). See {@link TimeUntilDisconnectNotification}.
      */
-    private void handleCommandReadinessNotification(final TimeUntilDisconnectNotification timeUntilDisconnectNotification) {
-
-        System.out.println(String.format("Device is ready to receive a command : <%s>.", timeUntilDisconnectNotification.toString()));
-        // fill in specific code to e.g. try to send a command here
+    private void handleCommandReadinessNotification(final TimeUntilDisconnectNotification notification) {
+
+        System.out.println(String.format("Device is ready to receive a command : <%s>.", notification.toString()));
+
+        final String tenantId = notification.getTenantId();
+        final String deviceId = notification.getDeviceId();
+
+        honoClient.getOrCreateCommandClient(tenantId, deviceId).map(commandClient -> {
+            // generate a correlationId for the application in the payload
+            final UUID correlationIdForPayload = UUID.randomUUID();
+            final JsonObject jsonCmd = new JsonObject().put("brightness", (int)(Math.random() * 100)).
+                    put("correlationId", correlationIdForPayload.toString());
```

Comment:
Did we not wanted to show a direct case without an internal correlationId (for later events) as our first case? I think this might be confusing, now?!

Reply:
Yes, you are right, I will remove it!